src/watch.rs

```diff
@@ -105,7 +106,7 @@ fn build_watcher(mut watch_tx: Sender<DebouncedEvent>) -> Result<(JoinHandle<()>
     let (tx, rx) = std::sync::mpsc::channel();
     let mut watcher = watcher(tx, std::time::Duration::from_secs(1)).context("failed to build file system watcher")?;
     watcher
-        .watch(".", RecursiveMode::Recursive)
+        .watch(PathBuf::from(".").canonicalize().unwrap().as_path(), RecursiveMode::Recursive)
```

Comment:
```suggestion
        .watch(PathBuf::from(".").canonicalize().context("failed to canonicalize CWD")?.as_path(), RecursiveMode::Recursive)
```

Reply:
I misclicked and deleted your suggestion ahah. You're right, though and I should not have used unwrap()